addons/mail/static/src/js/composer.js

```diff
@@ -327,29 +318,60 @@ var MentionManager = Widget.extend({
         return validated_selection;
     },
 
-    // Cursor position and selection utils
-    _get_selection_positions: function () {
-        var el = this.composer.$input.get(0);
-        return el ? {start: el.selectionStart, end: el.selectionEnd} : {start: 0, end: 0};
-    },
-    _set_cursor_position: function (pos) {
-        this.composer.$input.each(function (index, elem) {
-            if (elem.setSelectionRange){
-                elem.setSelectionRange(pos, pos);
-            }
-            else if (elem.createTextRange){
-                elem.createTextRange().collapse(true).moveEnd('character', pos).moveStart('character', pos).select();
-            }
+    // Handlers
+    // on click mention in chatter
+    on_click_mention_item: function (event) {
```

Comment:
missing docstring, should be renamed _onClickMentionItem, should be in 'Handlers' section